client/src/pages/Lecture.js

```diff
@@ -1,76 +1,102 @@
 import { React, useState, useEffect } from 'react';
 import { Link, useParams, useNavigate } from 'react-router-dom';
-import { TailSpin } from  'react-loader-spinner'
+import { TailSpin } from 'react-loader-spinner'
 import Notice from '../components/Notice'
 import { Button, Card } from "react-bootstrap"
 import useLectureQuestions from "../hooks/useLectureQuestions";
 import useCourse from "../hooks/useCourse";
 import useLectures from '../hooks/useLectures';
 import QuestionCard from '../components/QuestionCard';
 
+import io from 'socket.io-client';
```

Comment:
`socket.io-client` appears to be imported, but is not added to the client package.json causing compilation to fail.